spec/models/short_link_spec.rb

```diff
@@ -47,12 +47,12 @@
   describe '#target=' do
     it 'tacks on http when scheme is missing' do
       build(:short_link, :target => 'google.com')
-        .target.should == 'http://google.com'
+      .target.should == 'http://google.com'
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.